docs/image-sizes.md

```diff
@@ -12,6 +12,16 @@ They have default dimensions, but you can override these either with code or in
 
 ![Media Settings](./assets/media-settings.png)
 
+### `content_width`
```

Comment:
All other instances are either the variable or the global. I think it's probably ok to just refer to this section as "Content Width" as it's a general concept in WP.